### DISCUSSION TITLE
GitHub Actions badges don’t render correctly on Mobile README

### DISCUSSION THREAD

**Discussion by gemmakbarlow** [community/community#137192] [Mobile]
<img src="https://github.com/user-attachments/assets/068df641-41cf-4f96-b1b6-4a7b66e8ee1e" width=300>

They do render correctly on web, and (similar) CircleCI badges render correctly. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.175.0 (139716900)
Device: iPhone 13 Pro (iOS 17.6.1)
</details>

**Comment by elisealix22** [comment]
Thank you so much for reporting this @gemmakbarlow ! We are tracking this internally and I added your report to our open issue. Let us know if you have any other issues or suggestions in the meantime!

**Reply by gemmakbarlow** [reply]
Sounds great, thanks @elisealix22
